backend/src/test/java/reviewme/member/domain/ReviewerGroupTest.java

```diff
@@ -19,32 +24,83 @@ class ReviewerGroupTest {
         Member sancho = new Member("산초", 1);
         String groupName = "a".repeat(100);
         String description = "a".repeat(50);
-        LocalDateTime createdAt = LocalDateTime.of(2024, 7, 17, 12, 0);
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of(new GithubId(3));
 
         // when, then
-        assertDoesNotThrow(() -> new ReviewerGroup(sancho, List.of(new GithubId(3)), groupName, description, createdAt));
+        assertDoesNotThrow(
+                () -> new ReviewerGroup(sancho, githubIds, groupName, description, createdAt));
     }
 
     @ParameterizedTest
     @ValueSource(ints = {0, 101})
-    void 리뷰_그룹_이름_길이_제한을_벗어나는_경우_예외를_발생한다(int length) {
+    void 리뷰_그룹_이름_길이_제한을_벗어날_수_없다(int length) {
         // given
         String groupName = "a".repeat(length);
-        Member sancho = new Member("산초", 1);
-        LocalDateTime createdAt = LocalDateTime.of(2024, 7, 17, 12, 0);
+        Member sancho = 회원_산초.create();
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of();
+
         // when, then
-        assertThatThrownBy(() -> new ReviewerGroup(sancho, List.of(), groupName, "설명", createdAt))
+        assertThatThrownBy(() -> new ReviewerGroup(sancho, githubIds, groupName, "설명", createdAt))
                 .isInstanceOf(InvalidGroupNameLengthException.class);
     }
 
     @Test
-    void 리뷰_그룹_설명_길이_제한을_벗어나는_경우_예외를_발생한다() {
+    void 리뷰_그룹_설명_길이_제한을_벗어날_수_없다() {
         // given
         String description = "a".repeat(51);
-        Member sancho = new Member("산초", 1);
-        LocalDateTime createdAt = LocalDateTime.of(2024, 7, 17, 12, 0);
+        Member sancho = 회원_산초.create();
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of();
+
         // when, then
-        assertThatThrownBy(() -> new ReviewerGroup(sancho, List.of(), "그룹 이름", description, createdAt))
+        assertThatThrownBy(() -> new ReviewerGroup(sancho, githubIds, "그룹 이름", description, createdAt))
                 .isInstanceOf(DescriptionLengthExceededException.class);
     }
+
+    @Test
+    void 리뷰어_목록에_리뷰이가_들어갈_수_없다() {
+        // given
+        Member member = 회원_산초.create();
+        String groupName = "Group";
+        String description = "Description";
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> reviewerGithubIds = List.of(member.getGithubId());
```

Comment:
just 제안) reviewee를 나타내는 Member 객체가 sancho, member, reviewee 등으로 다양하게 변수명이 정의되어있는데 reviewee로 통일해봐도 좋을 것 같아요!

예를들어 이곳의 reviewerGithubIds 변수가, reviewee의 아이디를 갖고 reviewerGithubIds를 만드는 상황이라는 게 더 직관적으로 보일 것 같아서요~

```suggestion
        List<GithubId> reviewerGithubIds = List.of(reviewee.getGithubId());
```

Reply:
통일 완료!